string-hash/string-hash-tests.ts

```diff
@@ -0,0 +1,15 @@
+/// <reference path="index.d.ts" />
+
+import * as hash from "string-hash";
+
+const newHash: number = hash("Mary had a little lamb."); // 1766333550
+
+if (newHash === 1766333550) {
```

Comment:
Don't need the rest of the tests -- it takes a string, it returns a number, you're done.